Player Guides (#5)
FabricMC/fabric-docs: players/installing-mods.md

---
title: Installing Mods
description: A step by step guide on how to install mods for Fabric.
---

# Installing Mods

This guide will walk you through installing mods for Fabric using the Minecraft Launcher.

For third party launchers, you should consult their documentation.

## 1. Download the Mod

::: warning

> You should only download mods from sources you trust, for more information on finding mods, see the [Finding Mods](./finding-mods.md) guide.

Reviewer: ```suggestion
You should only download mods from sources you trust. For more information on finding mods, see the [Finding Mods](./finding-mods.md) guide.
```

:::

> The majority of mods require [Fabric API](https://modrinth.com/mod/fabric-api) as well, so you should download that too.

Reviewer 2: Link both Curseforge and Modrinth please.

When downloading mods, ensure that:

> - They work on the version of Minecraft you want to play on.

Reviewer: ```suggestion
- They work on the version of Minecraft you want to play on. A mod that works on version 1.20, for example, might not work on version 1.20.2.
```

> - They are for Fabric, not NeoForge, Rift, or any other mod loader.

Reviewer 2: ```suggestion
- They are for Fabric and not another mod loader.
```

> - They are for Fabric, not NeoForge, Rift, or any other mod loader.

Reviewer: ```suggestion
- They are for Fabric, not NeoForge, Spigot/Paper, or any other mod loader.
```

No version in which we support, had a working Rift release from what I can remember.

- Furthermore, they are for the correct edition of Minecraft (Java Edition).

## 2. Move the mod to the `mods` folder.

> The mods folder can be found in the following locations for each operating system:

Reviewer: Explain how to open this directory.
